modules/core/src/main/java/org/apache/ignite/internal/ClusterMetricsSnapshot.java

```diff
@@ -823,12 +823,12 @@ public void setCurrentIdleTime(long curIdleTime) {
 
     /** {@inheritDoc} */
     @Override public float getBusyTimePercentage() {
-        return 1 - getIdleTimePercentage();
+        return Math.max(100 - getIdleTimePercentage(), 0);
     }
 
     /** {@inheritDoc} */
     @Override public float getIdleTimePercentage() {
-        return getTotalIdleTime() / (float)getUpTime();
+        return Math.min((float)100. * getTotalIdleTime() / getUpTime(), (float)100.);
```

Comment:
Let's remove a multiplier here to leave public contract unchanged.

Reply:
Again, the method is called " Percentage"